src/components/structures/RoomView.js

```
@@ -865,7 +865,14 @@ module.exports = React.createClass({
             action: 'leave_room',
             room_id: this.props.roomId,
         });
-        this.props.onFinished();        
+    },
+
+    onForgetClick: function() {
+        MatrixClientPeg.get().forget(this.props.roomId).done(function() {
+            dis.dispatch({ action: 'view_next_room' });
+        }, function(err) {
+            console.error("Failed to forget room: %s", err);
```

why would we swallow this error msg? can we ErrorDialog it please?